WebRoot/js/zimbraAdmin/accounts/controller/ZaAccountListController.js

```diff
@@ -912,28 +912,28 @@ function (listArr) {
 				break;
 			}
 			dlgMsg += "<li>";
-			var szAccName = listArr[key].attrs[ZaAccount.A_displayname] ? AjxStringUtil.htmlEncode(listArr[key].attrs[ZaAccount.A_displayname]) : listArr[key].name;
+			var szAccName = listArr[key].attrs[ZaAccount.A_displayname] ? listArr[key].attrs[ZaAccount.A_displayname] : listArr[key].name;
             if(szAccName.length > 50) {
                 var beginIx = 0;
                 var endIx = 50;
 				do {
```

Comment:
fixed by using tabs in the entire function
i prefer spaces instead of tabs, but as admin console in majority place uses tabs so going with that